go/obscuronode/enclave/enclave.go

```diff
@@ -540,6 +615,13 @@ func NewEnclave(
 		nodecommon.LogWithID(obscurocommon.ShortAddress(nodeID), "validateBlocks is set to false. L1 blocks will not be validated.")
 	}
 
+	var attestationProvider AttestationProvider
+	if attestation {
+		attestationProvider = &EgoAttestationProvider{}
+	} else {
+		attestationProvider = &DummyAttestationProvider{}
```

Comment:
let's add a warning here that there is no attestation.
Similar to what is in place for `validateBlocks`
To try to prevent misconfiguration